src/test/java/DukeTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+    @Test
+    public void ToDoTest(){
```

Comment:
This JUnit test should be in its own test class.

eg.
public class ToDoTest {
... methods to test ToDo
}

This class should be for testing Duke in isolation.